packages/x-date-pickers/src/locales/utils/pickersLocaleTextApi.ts

```diff
@@ -60,6 +60,25 @@ export interface PickersLocaleText<TDate> {
   fieldMeridiemPlaceholder: () => string;
 }
 
+export interface PickersLocaleText<TDate>
+  extends PickersComponentAgnosticLocaleText<TDate>,
+    PickersComponentSpecificLocaleText {}
+
 export type PickersInputLocaleText<TDate> = Partial<PickersLocaleText<TDate>>;
 
+/**
+ * Translations that can be provided directly to the picker components.
+ * It contains some generic translations like `toolbarTitle`
+ * which will be dispatched to various translations keys in `PickersLocaleText`, depending on the pickers received them.
+ */
+export interface PickersInputComponentLocaleText<TDate>
+  extends Partial<PickersComponentAgnosticLocaleText<TDate>> {
+  toolbarTitle?: string;
```

Comment:
What do you think about adding some JSDoc to allow for better DX discoverability/clarity?

```suggestion
  /**
   * Provide explicit title for the toolbar per component
   */
  toolbarTitle?: string;
```

Reply:
I added some JSDoc :+1: